### PR TITLE
Add docstrings to some methods and functions.

### PR DESCRIPTION
**Pull request by lucc**
I randomly picked some of the places where it was easy to make out what to write.  Please have a look if I captured the semantics of the code correctly.

### CODE REVIEW

File: alot/account.py [hunk 1]
```diff
@@ -20,6 +20,7 @@ class StoreMailError(Exception):
 
 
 class Account(object):
+
```
Comment (pazz) [inline alot/account.py]: why do you insert empty new lines here (and below)?

Comment (lucc) [inline alot/account.py]: The templates I use when writing new code have that so I'm used to it... Will remove them.

Comment (lucc) [inline alot/account.py]: What about the empty line below the docstring? (I assume you meant the other docstring when you said "below")

Comment (pazz) [inline alot/account.py]: yes, I meant the other docstring. I don't really mind, just wanted to make sure we stick to one convention.

Comment (lucc) [inline alot/account.py]: I fixed the newline above the docstrings so you can unblock this.

**Comment by pazz** [comment]
thanks

**Pull request merged**